### ISSUE TITLE
Not in npm registry?

### ISSUE THREAD

**Issue by jorgschulze73** [roschaefer/jsonresume-theme-universal#1]
Installation guide mentions to install package via `npm install jsonresume-theme-universal` but it seems that such package does not exist in npm registry:
```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/jsonresume-theme-universal - Not found
npm ERR! 404
npm ERR! 404  'jsonresume-theme-universal@*' is not in this registry.
npm ERR! 404
```

**Comment by roschaefer** [comment]
Hey @jorgschulze73 thanks for the heads up!

This theme is not yet published, I just updated the `README.md`. Did you find the Github repo through a search engine?

I recently found [this repo](https://github.com/oguzgelal/resume) which is very much doing the same thing but without [json resume](https://jsonresume.org/) and using a custom config format instead. They use [Next.js](https://nextjs.org/) as a build tool. Now I ask myself if it's not better just to write a conversion tool for json resume file format? What do you think?

**Comment by jorgschulze73** [comment]
Yes, I found this repo while searching for some cool, recently updated themes for my resume.
Honestly it's your call if you want to integrate with JSON Resume or if you prefer to just write a conversion tool for the project you linked; if you ask me, JSON Resume seems to target a bigger audience cause I assume many people would prefer a generic, extensible solution over a custom one.

**Comment by roschaefer** [comment]
Here you go: https://github.com/roschaefer/resume/tree/jsonresume

Here's how you can try it out:
https://github.com/oguzgelal/resume/pull/2

I'm pretty sure we can get best of both worlds, we just need to export a library that can be consumed by `resume-cli`.
